huaweicloud/resource_huaweicloud_cce_cluster_v3.go

```diff
@@ -147,6 +147,10 @@ func resourceCCEClusterV3() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"kube_config_raw": {
```

Comment:
Add `StateFunc` function will be better.

Reply:
It seems the StateFunc won't be called until d.Set( ) succeed，but r.Body can't be set directly as TypeString